fluent-fallback/src/errors.rs

```diff
@@ -2,7 +2,7 @@ use fluent_bundle::FluentError;
 use std::error::Error;
 use unic_langid::LanguageIdentifier;
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
Question (blocking): Similar to above, I'd like to know what this clippy rule is before merging, since it's changing a public facing API.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq

This is more of a judgement call tbh, is there a reason to limit it to `PartialEq` and not `Eq`?